conf/iCubFindDependencies.cmake

```diff
@@ -64,7 +64,7 @@ checkandset_dependency(IPOPT)
 checkandset_dependency(OpenCV)
 checkandset_dependency(Qt5)
 
-set(MINIMUM_REQUIRED_icub_firmware_shared_VERSION 1.39.1)
+set(MINIMUM_REQUIRED_icub_firmware_shared_VERSION 1.40.1)
```

Comment:
The new version is 1.41.1 and it's already integrated so just remove this modification.